straxen/contexts.py

```diff
@@ -50,6 +50,7 @@
 # these are placeholders to avoid calling cmt with non integer run_ids. Better solution pending.
 # s1,s2 and fd corrections are still problematic
 xnt_simulation_config = deepcopy(xnt_common_config)
+# This gain model is the average to_pe. For something more fancy use the CMT
```

Comment:
Do we need this comment I mean it does not even fully fit. e.g. MV and elif.

Reply:
the place I moved it from it made even less sense. I think it's Diegos or Peters. I can remove it